src/external/pybind11/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+project(TuriExternalDependencies)
```

Comment:
What's the purpose of this line/file?

Reply:
Everything inside `src/external` had a CMakeLists.txt. I made one for pybind11 to keep it consisitent.